common/persistence/src/main/resources/db/migration/V23__createDccRevocationTables.sql

```diff
@@ -0,0 +1,14 @@
+CREATE TABLE revocation_entry (
+    kid bytea NOT NULL,
+    type bytea NOT NULL,
+    hash bytea NOT NULL,
+    PRIMARY KEY(kid, type, hash)
+);
+
+CREATE TABLE revocation_etag (
+    path varchar(255) PRIMARY KEY,
+    etag varchar(255) NOT NULL
+);
```

Comment:
is this used already somewhere?

Reply:
not yet, this will be for further optimization, I just added this in the same migration script